applications/ContactStructuralMechanicsApplication/tests/NightlyTests.py

```diff
@@ -7,7 +7,7 @@
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import Kratos_Execute_Contact_Structural_Test as Execute_Test
 
-# This utiltiy will control the execution scope in case we need to acces files or we depend
```

Comment:
We can correct the "utiltiy" word if we correct access...

Reply:
Good point. Done.